packages/forms-web-app/src/validators/before-you-start/enforcement-notice.js

```diff
@@ -0,0 +1,17 @@
+const { body } = require('express-validator');
+
+const validEnforcementNoticeOptions = ['yes', 'no'];
+
+const ruleEnforcementNotice = () =>
+  body('enforcement-notice')
+    .notEmpty()
+    .withMessage('Select Yes if you have received an enforcement notice')
```

Comment:
The y in yes should be lowercase, according to the ticket.

Reply:
Corrected